app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -31,7 +31,7 @@
 
       function appendHintToFormLabel () {
         var $label = $($changeNotesSection).find('label')
-        var exampleText = 'For example, "Edited chapter 6 - centres in Cardiff and Aberystwyth have closed."'
+        var exampleText = 'Tell users what has changed, where and why. Write in full sentences, leading with the most important words. For example, "College A has been removed from the registered sponsors list because its licence has been suspended." <br> Read the <a href="https://www.gov.uk/guidance/content-design/writing-for-gov-uk#change-notes" target="_blank">guidance about change notes</a> (opens in a new tab).'
```

Comment:
This is as 'yuck' as you think it is! It doesn't feel right to be injecting hardcoded HTML into the page using JavaScript here.

I haven't delved too far down that 🐰🕳️ , but I'd hope there's a good reason why this is being done rather than it being rendered in the server-side HTML view.

If this is simply to work around the fact that the use of the `form.text_area` helper:

https://github.com/alphagov/whitehall/blob/963d8779decf00a8206aa7ffc403f08a2d5ba29b/app/views/admin/editions/_change_notes.html.erb#L38

doesn't support a `hint:` parameter, then I'd be tempted to manually add the the label and hint text rather than using the helper to create it. That might mean we could do away with the JS here, too.

Not sure if I've explained that very well – happy to talk it through.